docs/application/tizen-studio/release-notes/3-5-release-notes.md

```diff
@@ -0,0 +1,72 @@
+# Tizen Studio 3.5 Release Notes
+
+-  Release Date: Oct 30, 2019
+
+
+## IDE and Tools
+
+### New Features
+-   IDE
+	-   Oracle JDK/OpenJDK 12 support has been added to make the IDE and tools work with Java 12.
+	-   Project Creation Wizard has been improved to add Flat View. Flat View provides a different view while creating a project.
```

Comment:
Project Creation Wizard is not a proper noun, so can we change "Project Creation Wizard" to "Project creation wizard"?
If not then can we use the rephrase from the [previous comment](https://github.com/Samsung/tizen-docs/pull/845#discussion_r329434941)?

Please verify.

Reply:
Done.